terraform/gcp-kubernetes-cluster/networking.tf

```diff
@@ -0,0 +1,73 @@
+resource "google_compute_network" "network" {
+  name                            = "${var.cluster_name}-network"
+  routing_mode                    = "REGIONAL" # global
+  auto_create_subnetworks         = false
+  mtu                             = 1460
```

Comment:
where are you getting this value? perhaps point to the docs?

Reply:
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_network#routing_mode